classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,1215 @@
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method31(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method32(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method33(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method34(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method35(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method36(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method37(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method38(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method39(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method40(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method41(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method42(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method43(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method44(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method45(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method46(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method47(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method48(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method49(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method50(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method51(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method52(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method53(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method54(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method55(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method56(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method57(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method58(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method59(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method60(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method61(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method62(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method63(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method64(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method65(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method66(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method67(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method68(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method69(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method70(){
+		Integer i = 0;
```

Comment:
This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.